charts/tempo/values.yaml

```diff
@@ -12,7 +12,7 @@ annotations: {}
 
 tempo:
   repository: grafana/tempo
-  tag: 2.0.1
+  tag: 2.1.1
```

Comment:
We can set these 2 tags to null and modify the statefulset.yaml to also get the version from the chart AppVersion

```
image: {{ .Values.tempo.repository }}:{{ .Values.tempo.tag | default .Chart.AppVersion | quote }}
```
and
```
image: {{ .Values.tempoQuery.repository }}:{{ .Values.tempoQuery.tag | default .Chart.AppVersion | quote }}
```

Reply:
nice. totally agree with this.